docs/src/pages/components/data-grid/filtering/CustomRatingFilterOperator.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { Rating } from '@material-ui/lab';
+import {
+  PreferencePanelsValue,
+  DataGrid,
+  FilterInputValueProps,
+  NUMERIC_OPERATORS,
+} from '@material-ui/data-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+
+const useStyles = makeStyles({
+  ratingContainer: {
+    display: 'inline-flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    height: 48,
+    paddingLeft: 20,
+  },
+});
+
+function RatingInputValue(props: FilterInputValueProps) {
+  const classes = useStyles();
+  const { item, applyValue } = props;
+  const [filterValueState, setFilterValueState] = React.useState<number>(
+    Number(item.value),
+  );
+
+  const onFilterChange = React.useCallback(
+    (event) => {
```

Comment:
Per convention & consistency

```suggestion
  const handleFilterChange = (event) => {
```